python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -346,6 +346,10 @@ def save(
                 tf.squeeze(dense_layer.get_weights()[0]).numpy())
             ),
             columns=["feature", "weight"])
+        # Adding log for bias value
+        if len(dense_layer.get_weights())>1:
+            bias_val = dense_layer.get_weights()[1][0]
+            linear_model_coefficients.loc[len(linear_model_coefficients.index)] = ['bias', bias_val]
```

Comment:
It is strange that we are hard-coding a name like `bias` in this low-level code. What is the effect of these lines of code before/after their addition to the dataframe `linear_model_coefficients`? I have not used this part of the library yet so this may be a silly comment.

Reply:
Makes sense to me. We do not use bias in our training because of how the model is deployed, but this looks like worth having.